### PR TITLE
cleaning up registerPatientTest

### PR DESCRIPTION
**Pull request by HerbertYiga**
This pr cleans up the registerPatientTest and makes sure the test works as expected

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/RegisterPatientSteps.java [hunk 1]
```diff
@@ -34,19 +34,20 @@ public void destroy() {
 
 	@Before(RunTest.HOOK.REGISTRATION)
 	public void setLoginPage() {
+		System.out.println(".... User"
+				+ " Login......");
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/RegisterPatientSteps.java:38]: Hopefully you will correct these changes

**Comment by HerbertYiga** [comment]
hi @mozzy11 this looks good enough locally,you could look through here!

**Comment by mozzy11** [comment]
@HerbertYiga ,
you can set the image [push](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/.github/workflows/publish.yml#L43) to false so that we dont have failing PRs because of faling to publish an image

**Comment by HerbertYiga** [comment]
> you can set the image [push](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/.github/workflows/publish.yml#L43) to false so that we dont have failing PRs because of faling to publish an image

ok then

**Comment by HerbertYiga** [comment]
> you can set the image [push](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/.github/workflows/publish.yml#L43) to false so that we dont have failing PRs because of faling to publish an image

hi @mozzy11  fixed this

**Pull request merged**